client/src/ui/organisms/placement/index.tsx

```diff
@@ -394,6 +418,64 @@ function RenderHpPlacement({
   );
 }
 
-function RenderBottomBanner({ place }: PlacementRenderArgs) {
-  return <div ref={place} className="empty-place bottom-banner"></div>;
+function RenderBottomBanner({
+  place,
+  extraClassNames = [],
+  click,
+  image,
+  imageWidth,
+  imageHeight,
+  copy,
+  user,
+  style,
+  version = 1,
+  typ,
+}: PlacementRenderArgs) {
+  return (
+    <div className="bottom-banner-container" style={style}>
+      <section
+        ref={place}
+        className={["place", "bottom-banner", ...extraClassNames].join(" ")}
+      >
+        <a
+          className="pong"
+          data-glean={`pong: pong->click ${typ}`}
+          href={`/pong/click?code=${encodeURIComponent(
+            click
+          )}&version=${version}`}
+          target="_blank"
+          rel="noreferrer"
+        >
+          <img
+            src={`/pimg/${encodeURIComponent(image || "")}`}
+            alt={copy}
+            width={imageWidth}
+            height={imageHeight}
+          ></img>
+        </a>
+        <a
+          href="/en-US/advertising"
```

Comment:
Nit: Why `en-US`, not `${locale}`? The page is available in other locales: https://developer.allizom.org/fr/advertising